client/src/components/Modal/VideoModal.component.jsx

```diff
@@ -38,6 +52,13 @@ function VideoModal({ description, thumbnail, src }) {
           </span>
           <span style={{ fontSize: 12 }}>video</span>
         </div>
+        {role === "superAdmin" && (
+          <div className="v-delete-icon">
```

Comment:
use `className="mt-2 position-absolute"` instead of `position: absolute;
  right: 5px;` in the class